packages/client/src/components/interface/Navigation.tsx

```diff
@@ -361,7 +361,13 @@ const NavigationView = (props: IFullProps) => {
     (IS_PROD_ENVIRONMENT && window.config.STATISTICS_DASHBOARD_URL)
 
   React.useEffect(() => {
-    if (!userDetails || !loadWorkqueueStatuses) {
+    if (
+      !userDetails ||
+      !loadWorkqueueStatuses ||
+      userDetails.systemRole === 'LOCAL_SYSTEM_ADMIN' ||
+      userDetails.systemRole === 'NATIONAL_SYSTEM_ADMIN' ||
+      userDetails.systemRole === 'PERFORMANCE_MANAGEMENT'
```

Comment:
https://github.com/opencrvs/opencrvs-core/blob/ed2089a2f696245e26c520e9388a354d9cecbec5/packages/client/src/views/SysAdmin/SysAdminContentWrapper.tsx#L233
think we could utilize the loadWorkqueueStatuses variable and use `loadWorkqueueStatuses={false}` here